### ISSUE TITLE
fps argument is ignored when rendering animation

### ISSUE THREAD

**Issue by minimaxir** [thomasp85/gganimate#316]
Also occurs with `av_renderer()`.

```r
library(ggplot2)
library(gganimate)

anim <- ggplot(mtcars, aes(factor(cyl), mpg)) + 
  geom_boxplot() + 
  transition_states(
    gear,
    transition_length = 2,
    state_length = 1
  ) +
  enter_fade() + 
  exit_shrink() +
  ease_aes('sine-in-out')

anim_save(
  'test.mp4',
  anim,
  renderer = ffmpeg_renderer(format = ".mp4"),
  fps=60
)
```

Output is 25 fps regardless of the fps parameter.

ffmpeg output:

```
Input #0, image2, from '/var/folders/8b/fs0_fz9s00v1hync3vg4d2tm0000gn/T//RtmpwX3ytz/186928768126/gganim_plot%4d.png':
  Duration: 00:00:04.00, start: 0.000000, bitrate: N/A
    Stream #0:0: Video: png, rgba(pc), 480x480, 25 fps, 25 tbr, 25 tbn, 25 tbc
```

Codec output:

<img width="502" alt="Screen Shot 2019-05-01 at 8 42 12 PM" src="https://user-images.githubusercontent.com/2179708/57055759-4dce6200-6c52-11e9-8850-91445255bb2c.png">

On latest R, gganimate, and ffmpeg.

**Comment by minimaxir** [comment]
This apparently also occurs with the default `gifski_renderer()` (only 100 frame generated for a 4 second animation). Editing title.

**Comment by minimaxir** [comment]
Also occurs with vanilla `animate`.

**Comment by minimaxir** [comment]
Workaround which gets the desired behavior (render at 25fps and speed it up appropriately)

```r
anim_save(
  'test.mp4',
  anim,
  renderer = ffmpeg_renderer(format = ".mp4"),
  fps=60,
  duration=4
)

system("ffmpeg -y -i test.mp4 -r 60 -filter:v 'setpts=25*PTS/60' test2.mp4")
```

And after a bit of [video-to-gif cleaning](https://cassidy.codes/blog/2017/04/25/ffmpeg-frames-to-gif-optimization/):

![test2](https://user-images.githubusercontent.com/2179708/57058518-4bc1ce80-6c65-11e9-8f16-0fb72882436c.gif)

Not shabby.

**Comment by thomasp85** [comment]
Fixed by 3c71cea5

I could only replicate your findings for ffmpeg_renderer — the other worked as expected. I discovered I was passing in the frame rate in the wrong way when constructing the call
